Deployment/Deploy.azcli

```diff
@@ -18,6 +18,10 @@ $windowsSampleFunctionPythonName = "BenchMark-Sample-Win-Python";
 $linuxSampleFunctionCsharpName = "BenchMark-Sample-Lin-Csharp";
 $linuxSampleFunctionNodejsName = "BenchMark-Sample-Lin-Nodejs";
 $linuxSampleFunctionPythonName = "BenchMark-Sample-Lin-Python";
+$windowsSampleFunctionCsharpName = "BenchMark-Sample-Win-Csharp-3";
```

Comment:
Ik was deze in een andere branch aan het overnemen, en nu zie ik dat deze naam identiek is aan bovenstaande voor Csharp.
wordt deze dus niet overschreven?